es2-persistence/src/main/java/com/netflix/conductor/dao/es/index/ElasticSearchDAO.java

```diff
@@ -500,4 +501,25 @@ public String get(String workflowInstanceId, String fieldToGet) {
 		}
 		return ids;
 	}
+
+	//copy paste from com.netflix.conductor.dao.es.index.ElasticSearchDAO5.searchRecentIncompletedWorkflows
+	public List<String> searchRecentIncompletedWorkflows(String indexName, long modifiedHoursAgo) {
```

Comment:
should use indexName that is already populated from configuration, otherwise caller has to get it from configuration again.